caffe-grid/pom.xml

```diff
@@ -42,6 +42,11 @@
   </profiles>
   <dependencies>
     <dependency>
+      <groupId>net.sf.py4j</groupId>
+      <artifactId>py4j</artifactId>
+      <version>0.8.1</version>
+    </dependency>
+    <dependency>
```

Comment:
We don't need this dependency any more. It works fine w/o it on my local machine at least.